modules/caddyhttp/encode/zstd/zstd.go

```diff
@@ -27,7 +30,9 @@ func init() {
 }
 
 // Zstd can create Zstandard encoders.
-type Zstd struct{}
+type Zstd struct {
+	Level zstd.EncoderLevel `json:"level,omitempty"`
```

Comment:
I think we should add a godoc comment here to explain the possible options.

Reply:
Generally we should built-in or caddy defined type to configure a plugin. We should use string to specify the level and check if it's one of the possible values provided by zstd.

Reply:
`Level` should be just string. We can use a private member to specify the actual level used with type `zstd.EncoderLevel`.